modules/persistent-storage-local-install.adoc

```diff
@@ -34,6 +34,19 @@ To block the Local Storage Operator from inheriting the cluster-wide default sel
 $ oc annotate project openshift-local-storage openshift.io/node-selector=''
 ----
 
+. Optional: Allow local storage to run on the management pool of CPUs in single-node deployment.
++
+Use the Local Storage Operator in single-node deployments and allow the use of CPUs that belong to the `managment` pool. Perform this step on single-node installations that use management workload partitioning.
++
+To allow Local Storage Operator to run on the management CPU pool, run following commands:
++
+[source,terminal]
+----
+$ oc annotate namespace openshift-local-storage openshift.io/node-selector=''
```

Comment:
@lpettyjo 
I mean it should replace the previous existed Line34, not need to add here. (Sorry not clear in my last comment.)

Reply:
OK. Thanks! Fixed.

Reply:
Sorry but I still see this line present, it should be removed here.

Reply:
Fixed.